pkg/gadgets/trace/capabilities/tracer/tracer_test.go

```diff
@@ -142,18 +150,22 @@ func TestCapabilitiesTracer(t *testing.T) {
 			validateEvent: utilstest.ExpectOneEvent(func(info *utilstest.RunnerInfo, _ interface{}) *types.Event {
 				return &types.Event{
 					Event: eventtypes.Event{
-						Type: eventtypes.NORMAL,
+						Type:      eventtypes.NORMAL,
+						Timestamp: 1,
 					},
-					MountNsID: info.MountNsID,
-					Pid:       uint32(info.Pid),
-					UID:       uint32(info.UID),
-					Comm:      info.Comm,
-					Syscall:   "bind",
-					CapName:   "NET_BIND_SERVICE",
-					Cap:       10,
-					Audit:     1,
-					InsetID:   &false_,
-					Verdict:   "Allow",
+					MountNsID:     info.MountNsID,
+					Pid:           uint32(info.Pid),
+					UID:           uint32(info.UID),
+					Comm:          info.Comm,
+					Syscall:       "bind",
+					CapName:       "NET_BIND_SERVICE",
+					Cap:           10,
+					Audit:         1,
+					InsetID:       &false_,
+					Verdict:       "Allow",
+					CurrentUserNs: 1,
```

Comment:
We can also test this by using something like -> https://github.com/inspektor-gadget/inspektor-gadget/commit/6b490ea269c704240a3fd1162fe475403afc1d64. Our runner isn't unsharing the user ns, hence we can assume `TargetUserNs` is always equal to `CurrentUserNs` in these tests? 

I think we can do the same for CapEffective, I was checking https://pkg.go.dev/kernel.org/pub/linux/libs/security/libcap/cap#Set but didn't find a way to get the bitmask of the capabilities.

Reply:
> We can also test this by using something like -> https://github.com/inspektor-gadget/inspektor-gadget/commit/6b490ea269c704240a3fd1162fe475403afc1d64.

Thanks! I applied your patch.

> Our runner isn't unsharing the user ns, hence we can assume TargetUserNs is always equal to CurrentUserNs in these tests?

Probably: neither the runner nor the workload (generateEvent) create user namespaces. But I didn't change this. Do you want me to add a test for this?

> I think we can do the same for CapEffective, I was checking https://pkg.go.dev/kernel.org/pub/linux/libs/security/libcap/cap#Set but didn't find a way to get the bitmask of the capabilities.

We can probably do that with the gocapability package. I now use this in the capsNames() function. Could this be done in a follow-up PR?

Reply:
> But I didn't change this. Do you want me to add a test for this?

Yes, I think it's better to check if those are equal instead of relying that they are not empty. (If for any reason we mess up the tracer to read garbage the tests will pass anyway). 

> We can probably do that with the gocapability package. I now use this in the capsNames() function. Could this be done in a follow-up PR?

For sure, would you mind opening an issue to avoid forgetting about this?